### PR TITLE
Implented basic MQTT SYS topics for stats

### PR DESCRIPTION
**Pull request by debjeet-sarkar**

### CODE REVIEW

**Comment by mcollina** [comment]
Woow, this is amazing? Would you mind adding a quick unit test with http://npm.im/tape?

**Comment by debjeet-sarkar** [comment]
@mcollina, shall.

**Comment by debjeet-sarkar** [comment]
@mcollina, waiting for you to make an init :)

**Comment by mcollina** [comment]
Done. Would you mind to setup a unit a test, and lint using standard?

**Comment by debjeet-sarkar** [comment]
@mcollina  Added test case and lint, the process was not ending after all the assertions and I had to exit it after the last one. Am I missing out on something.

**Comment by mcollina** [comment]
you need to close all those `net`  servers when a test finishes.

**Comment by debjeet-sarkar** [comment]
@mcollina, closed all the net servers. Would you mind to review it once.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module.exports = require('./stats')
```
Comment (mcollina) [inline index.js]: this file is not needed IMHO, we can just use `stats.js`

File: stats.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    aedesInstance.on(event.name, event)
+  })
+
+  aedesInstance.once('closed', function () {
+    clearInterval(timer)
+    aedesEvents.forEach(function (event) {
+      aedesInstance.removeListener(event.name, event)
+    })
+  })
+}
```
Comment (mcollina) [inline stats.js]: I think we can change this from Mosca style, and just export a single function `wire`, something like:

```js
const aedes = require('aedes')()
require('aedes-stats')(aedes)
```

File: index.js (hunk 1 above)
Comment (debjeet-sarkar) [inline index.js]: cool :)

File: stats.js (hunk 2 above)
Comment (debjeet-sarkar) [inline stats.js]: cool, implemented :)

**Comment by debjeet-sarkar** [comment]
@mcollina , sounds great !! . You can merge and release :)

**Comment by debjeet-sarkar** [comment]
@mcollina , you removed me as a contributor sir ? Any specific reason or feedback ?

**Comment by mcollina** [comment]
I do not know what happened. It looks like I just pressed the merge button on github.

**Comment by debjeetsarkar** [comment]
@robertsLando , hey can you add me to the Mosca Js organization ? Looks like I updated my GitHub username and every references got messed up 😭

**Comment by robertsLando** [comment]
Hi @debjeetsarkar, I never heard about you, what are you interested in contribute to? @mcollina Are you ok with this?

**Comment by mcollina** [comment]
Sure, he originally wrote this module

**Comment by robertsLando** [comment]
@debjeetsarkar Sent an invite check your email :)

**Pull request merged**